src/widget/container.rs

```diff
@@ -55,6 +69,7 @@ where
         }
 
         let style = style.build();
+        let size = self.options.size.unwrap_or(size);
```

Comment:
Why would the `size` option trump the passed `size` param? And why should you have to pass a `size` param if it won't get used because there's a `size` option?

Reply:
Sorry, I should have commented on that. The idea was that if you want a container to fill available space, `size` is `None`, but if you want it to have a specific size (such as the inner child container in the example), you can provide it. An `Option` is not really a very clear way of modeling that. Maybe an enum of `FillSpace | FixedWidth | FixedHeight | FixedSize` would be better?

Reply:
I'll punt on this for now since I think future changes will make the right thing to do more clear (implementing more flex-box-y type layout).